com.unity.multiplayer.mlapi/Tests/Editor/IndexAllocatorTests.cs

```diff
@@ -0,0 +1,117 @@
+using NUnit.Framework;
+using UnityEngine;
+
+namespace MLAPI.EditorTests
+{
+    public class FixedAllocatorTest
+    {
+        [Test]
+        public void SimpleTest()
+        {
+            int pos;
+
+            var allocator = new IndexAllocator(20000, 200);
+            allocator.DebugDisplay();
+
+            // allocate 20 bytes
+            Assert.IsTrue(allocator.Allocate(0, 20, out pos));
+            allocator.DebugDisplay();
+            Assert.IsTrue(allocator.Verify());
```

Comment:
Maybe we need a test to ensure that allocator.Verify actually works correct. I think if there was just a single bug in the Verify function of the allocator all tests might pass and not verify the allocator correctly.

Just a suggestion not sure if it necessary, depends on our confidence on the "Verify" function.

Reply:
Resolving, trying to trim this down to needed changes :-)